plugins/base/src/main/kotlin/transformers/documentables/utils.kt

```diff
@@ -5,10 +5,12 @@ import org.jetbrains.dokka.model.Documentable
 import org.jetbrains.dokka.model.ExceptionInSupertypes
 import org.jetbrains.dokka.model.properties.WithExtraProperties
 
+@Suppress("TYPE_MISMATCH_WARNING_FOR_INCORRECT_CAPTURE_APPROXIMATION")
```

Comment:
Are these suppresses still needed? There are still quite many of them. If they are not needed, could you please remove them?